ibtk/include/ibtk/HierarchyMathOps.h

```diff
@@ -1155,6 +1186,26 @@ class HierarchyMathOps : public SAMRAI::tbox::DescribedClass
                     int src2_idx = invalid_index,
                     SAMRAI::tbox::Pointer<SAMRAI::pdat::SideVariable<NDIM, double> > src2_var = nullptr);
 
+    /*!
+     * \brief Compute dst = alpha grad coef1 (div src1)  + beta src2, the variable coefficient
```

Comment:
Can we call this the "grad-div" operator? (I am not sure if there is a widely used name for this guy, but it does show up in various contexts in addition to computing stresses.)